expected/16.0/extra/nogis.out

```diff
@@ -0,0 +1,154 @@
+(3 rows)
+
+--Testcase 40:
+SELECT "i", gm, gg, t FROM "types_PostGIS" WHERE gm = '0101000020e6100000bf72ce99fe763e40ed4960730ed84d40'::geometry;
+ i | gm | gg | t 
+---+----+----+---
+(0 rows)
+
+-- Insert PostGIS/GEOS BLOB, read SpatiaLite BLOB
+--Testcase 41: OK
+INSERT INTO "types_PostGIS" ( "i", gm, gg, t ) VALUES (2, decode('0101000020e6100000bf72ce99fe763e40ed4960730ed84d40', 'hex'),  decode('0101000020e6100000bf72ce99fe763e40ed4960730ed84d40', 'hex'), '{"genus": "Rhododendron", "taxon": "Rhododendron ledebourii"}');
+ERROR:  This PostGIS data type is supported by SpatiaLite, but FDW compiled without GIS data support
+HINT:  Data type: "public.geometry" in column "gm"
+--Testcase 42:
+ALTER FOREIGN TABLE "types_PostGIS" ALTER COLUMN "gm" TYPE bytea;
+--Testcase 43:
+ALTER FOREIGN TABLE "types_PostGIS" ALTER COLUMN "gg" TYPE bytea;
+--Testcase 44: OK
+SELECT "i", gm, gg, t FROM "types_PostGIS";
+ i |                                                             gm                                                             |                                                             gg                                                             |                                                                                     t                                                                                     
+---+----------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ 1 | \x0001e6100000bf72ce99fe763e40ed4960730ed84d40bf72ce99fe763e40ed4960730ed84d407c01000000bf72ce99fe763e40ed4960730ed84d40fe | \x0001e6100000bf72ce99fe763e40ed4960730ed84d40bf72ce99fe763e40ed4960730ed84d407c01000000bf72ce99fe763e40ed4960730ed84d40fe | {"genus": "Rhododendron", "taxon": "Rhododendron ledebourii", "natural": "shrub", "genus:ru": "Рододендрон", "taxon:ru": "Рододендрон Ледебура", "source:taxon": "board"}
+(1 row)
+
+--Testcase 45:
+CREATE FOREIGN TABLE "♂" (
+	id int4 OPTIONS (key 'true'),
+	"UAI" varchar(254),
+	"⌖" geometry,
+	geom geometry,
+	"t₀" date,
+	"class" text,
+	"URL" varchar(80)
+) SERVER sqlite_svr;
+--Testcase 47:
+ALTER FOREIGN TABLE "♂" ALTER COLUMN "⌖" TYPE bytea;
+--Testcase 48:
+ALTER FOREIGN TABLE "♂" ALTER COLUMN "geom" TYPE bytea;
+--Testcase 49:
+SELECT * FROM "♂";
+ id | UAI | ⌖ | geom | t₀ | class | URL 
+----+-----+---+------+----+-------+-----
+(0 rows)
+
+--Testcase 50:
+ALTER FOREIGN TABLE "♂" ALTER COLUMN "⌖" TYPE geometry;
+--Testcase 51:
+ALTER FOREIGN TABLE "♂" ALTER COLUMN "geom" TYPE geometry;
+--Testcase 52:
+SELECT * FROM "♂";
+ id | UAI | ⌖ | geom | t₀ | class | URL 
+----+-----+---+------+----+-------+-----
+(0 rows)
```

Comment:
This is TC about empty data set in context of `nogis` test. No problems here because no data, but spatial names column data types are declared.

Reply:
Fixed by comment, please verify.

Reply:
Normally, we would like to avoid empty data test case as much as possible.  
I think it is better to keep current empty data set test case, and add more test cases to insert data and then select.